### ISSUE TITLE
Job perpetually queued 

### ISSUE THREAD

**Issue by Gilks** [hashview/hashview#383]
When attempting to start a job, it appears the job gets stuck in a "Queued" state. There are no other jobs running at the time.

Here's some output from the running [hashview](https://gist.github.com/Gilks/6224a7e78707cb215bc15c9582ab4cb6).

Version info:
```
Hashview: Most recent version available on Github
Hashcat: v3.6.0
RVM: 1.29.3
OS: Linux cracker 4.4.0-104-generic #127-Ubuntu SMP Mon Dec 11 12:16:42 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux
```

Any ideas where I should begin diagnosing this? I've attempted to follow all instructions in the "My jobs are not running" [troubleshooting](https://github.com/hashview/hashview/wiki/Troubleshooting) section.

**Comment by i128** [comment]
What tasks are you assigning it?

**Comment by Gilks** [comment]
I've tried the following the dictionary and bruteforce tasks.

**Comment by i128** [comment]
Go to hashfiles (menu option in the upper right), how many hashes does it list?

**Comment by ccammilleri** [comment]
have you been able to verify that hashcat is working properly? Also, running the following command when starting the job my help identify whats happening: `tail -f control/outfiles/hcoutput_*`

**Comment by i128** [comment]
Jobs will perpetually stall for two reasons, 

1. The hashfile is empty 0/0 
1. The wordlist is empty 0/0. 

This will be fixed in the next release ( v0.7.4-beta)

**Comment by Gilks** [comment]
There's no excuse for never coming back here to give an update.

This may be a confirmed bug for empty hashfiles/wordlists but in my particular instance it appears hashcat would start and then quit due to an over heating graphics card (thanks @ccammilleri for the idea). We are currently determining the best course of action internally to get it replaced.

If there's still an issue after the new card is installed, I'll come back.
